src/Core/StatusResponse.php

```diff
@@ -32,11 +32,21 @@ public function amount()
         return arr_has($this->data, 'amount') ? $this->data['amount'] : -1;
     }
     
+    /**
+     * Get the reference of the transaction
+     *
+     * @return string
+     */
     public function reference()
     {
         return arr_has($this->data, 'reference') ? $this->data['reference'] : '';
     }
 
+    /**
+     * Get the paynowreference of the transaction
+     *
```

Comment:
Can you please give the fields a bit of a better description. You can use 

https://developers.paynow.co.zw/docs/status_update.html#statuses

as a startpoint